lib/vsc/ldap/group.py

```diff
@@ -76,8 +76,8 @@ def get_ldap_info(self):
         """Retrieve the data from the LDAP to initially fill up the ldap_info field."""
         group_ldap_info = self.ldap_query.group_filter_search("cn=%s" % (self.group_id))
         if len(group_ldap_info) == 0:
-            self.log.error("Could not find a group in the LDAP with the ID %s, raising NoSuchGroupError"
-                           % (self.group_id))
+            self.log.error("Could not find a group in the LDAP with the ID %s, raising NoSuchGroupError",
```

Comment:
you can replace `self.log` with logging everywhere

Reply:
sure, but then the logger name is lost.

Reply:
done

Reply:
the logger name, well, it's not a big loss imho. we used to do that to make debugging easier, but tbh, i have rarely relied on the logname while debugging